src/main/java/com/cloudbees/jenkins/plugins/bitbucket/PullRequestSCMHead.java

```diff
@@ -27,15 +27,25 @@
 import com.cloudbees.jenkins.plugins.bitbucket.api.BitbucketRepositoryType;
 import edu.umd.cs.findbugs.annotations.NonNull;
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import hudson.Extension;
 import java.io.ObjectStreamException;
-import jenkins.scm.api.mixin.ChangeRequestSCMHead;
+import jenkins.plugins.git.AbstractGitSCMSource;
 import jenkins.scm.api.SCMHead;
+import jenkins.scm.api.SCMHeadMigration;
+import jenkins.scm.api.SCMHeadOrigin;
+import jenkins.scm.api.SCMRevision;
+import jenkins.scm.api.mixin.ChangeRequestCheckoutStrategy;
+import jenkins.scm.api.mixin.ChangeRequestSCMHead2;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.DoNotUse;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
 
 /**
  * {@link SCMHead} for a BitBucket Pull request
+ *
  * @since FIXME
  */
-public class PullRequestSCMHead extends SCMHead implements ChangeRequestSCMHead {
+public class PullRequestSCMHead extends SCMHead implements ChangeRequestSCMHead2 {
```

Comment:
Roll on Java 8 as the baseline... when we can add new methods to an interface with a default implementation... then I get to deprecate the `ChangeRequestSCMHead2` and maybe move back